core/app.js

```diff
@@ -296,12 +296,18 @@ app.use(_.get("/employees",function *(){
   var isAdmin = yield auth.isAdmin(this.session.user);
   var client = db();
   var allAdmins = yield client.query("SELECT netid, email FROM users WHERE admin = TRUE;");
+  var names = []
+  for(var x = 0; x < allAdmins.rows.length; x++){
+    var db_id = (allAdmins.rows[x]).netid;
+    var data = yield user.ldapInfo(db_id);
+    names.push(data[db_id].cn);
+  }
```

Comment:
Let me try that again:

`var names = (yield user.ldapInfo.call(null, allAdmins.rows.map(function (a){return a.netid})).map(function(u){ return u.cn});`

This could be made easier to read with:

```javascript
var netids = allAdmins.rows.map(function (a){return a.netid});
var ldapDetails = yield user.ldapInfo.apply(null, netids);
var names = ldapDetails.map(function(u){ return u.cn; });
```